bin/shell-lint.sh

```diff
@@ -0,0 +1 @@
+../shell-lint.sh
```

Comment:
Hmm...it looks like we'll have to change `update-symlinks` to exclude this. We might want to pass a `-mindepth` option to one of the `find` commands in that script.

Reply:
For that matter `update-symlinks` ought to exclude itself too